charts/rancher-backup/rancher-backup/1.0.400+up1.0.4-rc4/templates/serviceaccount.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: {{ include "backupRestore.serviceAccountName" . }}
-  namespace: {{ .Release.Namespace }}
-  labels:
-    {{- include "backupRestore.labels" . | nindent 4 }}
```

Comment:
This file is marked as deleted by Git, but you can find the corresponding file is actually marked as created below.